examples/deployment-with-init/provider.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_version = ">= 0.12"
+
+  required_providers {
+    ec = {
+      source = "elastic/ec"
+    }
+  }
+}
+
+provider "ec" {
+  apikey = "azFXa0pIUUJaeWRjMmUzVXpCOG06VUJ1NEtFREtSNE9BMThxeG9mM1A3dw=="
```

Comment:
I'm not sure having an API key here is a good idea? Can we substitute it for `[REDACTED]` or something like that?

Reply:
😱

Reply:
I think we can just leave it as is, and ask users to export `EC_API_KEY` with the value in the markdown document.

Reply:
I've invalidated it either way, but opted to remove it and replace with <your API key here>